var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -30,7 +30,89 @@ class Tasmanian(CMakePackage):
     ApproximatioN is a robust library for high dimensional integration and
     interpolation as well as parameter calibration."""
 
-    homepage = "http://tasmanian.ornl.gov"
-    url      = "http://tasmanian.ornl.gov/documents/Tasmanian_v5.0.zip"
+    homepage = 'http://tasmanian.ornl.gov'
+    url      = 'https://github.com/ORNL/TASMANIAN/archive/v5.1.tar.gz'
 
-    version('5.0', '4bf131841d786033863d271739be0f7a')
+    version('develop', branch='master',
+            git='https://github.com/ORNL/TASMANIAN.git')
+
+    version('5.1', '5d904029a24470a6acf4a87d3339846e',
+            url='https://github.com/ORNL/TASMANIAN/archive/v5.1.tar.gz')
+
+    version('5.0', '4bf131841d786033863d271739be0f7a',
+            url='http://tasmanian.ornl.gov/documents/Tasmanian_v5.0.zip')
+
+    depends_on('cmake@3.5.0:', type='build')
+
+    variant('xsdkflags', default=False,
+            description='enable XSDK defaults for Tasmanian')
```

Comment:
I'm not sure how I feel about flags that enable "all" features. For example, if there were a package that depended on this one, if it needed to be built with MPI support, it would have to say "depends on `tasmanian+mpi` _or_ `tasmanian+xsdkflags`". I don't think there's a way to say that in the spec syntax right now.

Reply:
`xsdkflags` relates to the xSDK project, we are a collection of libraries that use a similar build systems and conform to common set of policies. MPI is always enabled within the xSDK project and that is the only time anyone would ever have to use `+xsdkflags`. 

That is, if anyone outside of xSDK needs Tasmanian, then they will have to use `+mpi`, the xSDK project meta package will be the only thing using `+xsdkflags`.